client/jira.go

```diff
@@ -103,6 +114,41 @@ func CommentIssue(issueId, comment string) (domain.Issue, error) {
 	return issue, nil
 }
 
+func DoTransition(issueId, transitionId string) error {
+	path := fmt.Sprintf("/rest/api/2/issue/%s/transitions", issueId)
+	doTransitionRequest := prepare(transitionId)
+	b, err := json.Marshal(doTransitionRequest)
+	request, err := constructPostRequest(path, string(b))
+
+	log.Println(request)
+	log.Println(string(b))
+
+	log.Printf("issueID %s - URL %s - transitionID %s", issueId, request.URL, transitionId)
```

Comment:
Can we remove these prints for the actual PR, please? :( Ideally we should put it in a logger with a `Debug` option so some of these can stay in, but until then it'd be good to declutter everything.